raspicheck.py

```diff
@@ -0,0 +1,69 @@
+import discord
+from discord.ext import commands
+from __main__ import set_cog, send_cmd_help, settings
+import asyncio
+import psutil

```

Comment:
actually, the new, new way to do it would be to just import it and put it in the cog's info.json requirements